src/tests/gov/nasa/jpf/StaticCallToNonStaticTest.java

```diff
@@ -0,0 +1,35 @@
+package gov.nasa.jpf;
+
+import org.junit.AfterClass;
+import gov.nasa.jpf.util.test.TestJPF;
+import org.junit.jupiter.api.Test;
+import java.io.File;
+
+
+
+
+public class StaticCallToNonStaticTest extends TestJPF {
+
+    @Test
+    public void testStaticCallToNonStatic() {
+  D.m();
```

Comment:
This test should expect an exception: Try
`Assertions.assertThrows(java.lang.IncompatibleClassChangeError, D.m());`
This should make the test fail, so we can ensure that the fix that we need is tested against.
After the fix is in place, the test will succeed.